config/config.go

```diff
@@ -145,6 +148,11 @@ func mergeConfigFromFile(configpath string, source Local) (Local, error) {
 
 	err = loadConfig(f, &source)
 
+	// If the PublicAddress in config file has the PlaceholderPublicAddress, treat it as if it were empty
+	if source.PublicAddress == PlaceholderPublicAddress {
```

Comment:
@algorandskiy can we live with this addressing the public address concern?

Reply:
looks good